packages/perspective-viewer-d3fc/src/js/legend/legend.js

```diff
@@ -92,6 +98,8 @@ function legendComponent(scrollLegend, scaleModifier) {
                 .select("path")
                 .style("fill", d => (isHidden(d) ? null : color(d)))
                 .style("stroke", d => (isHidden(d) ? null : withoutOpacity(color(d))));
+
+            cells.append("title").html(d => `${d}`);
```

Comment:
Were you converting to a string?
```suggestion
            cells.append("title").html(d => d);
```

Reply:
😆 oops.